src/libs/fileDownload/index.android.ts

```diff
@@ -46,7 +43,7 @@ function handleDownload(url, fileName) {
         const isLocalFile = url.startsWith('file://');
 
         let attachmentPath = isLocalFile ? url : undefined;
-        let fetchedAttachment = Promise.resolve();
+        let fetchedAttachment: Promise<void> | StatefulPromise<FetchBlobResponse> = Promise.resolve();
```

Comment:
Do we need `StatefulPromise<FetchBlobResponse>`? Maybe just `Promise<void | FetchBlobResponse>` would be enough?